src/mapeo-project.js

```diff
@@ -1079,16 +1104,29 @@ export class MapeoProject extends TypedEmitter {
     const missingAttachments = []
     // Attachments
     if (attachments) {
-      const mediaFolder = this.#exportPrefix('Media') + '/'
+      const mediaFolder = (await this.#exportPrefix('Media')) + '/'
       for (const attachment of seenAttachments.values()) {
-        const blobId = await this.#tryGetBlobId(attachment)
-        if (blobId === null) {
+        const ref = await this.#tryGetAttachmentBlob(attachment)
+        if (ref === null) {
+          missingAttachments.push(attachment)
+          continue
+        }
+
+        const { blobId, mimeType } = ref
+        const extension = mime.getExtension(mimeType)
+
+        if (!extension) {
           missingAttachments.push(attachment)
+          this.#l.log('Got unknown mime type in attachment blob', attachment)
```

Comment:
similarly here I kind of think we should still include the attachment even if we don't know the mimetype.